examples/layer-browser/src/examples/core-layers.js

```diff
@@ -209,14 +209,29 @@ const GridCellLayerExample = {
   }
 };
 
+function getMean(pts, key) {
+  const filtered = pts.filter(pt => Number.isFinite(pt[key]));
+
+  return filtered.length ?
```

Comment:
> if value is null or undefined, we do not include it when calculate Mean, so we need to get the total number of elements that are not null to divide the sum by

Then the comparison should not use `filtered.length`, but just `filtered` right?  Perhaps a good opportunity to add a test case that triggers this bug, since you are working on coverage anyway?